.changeset/mighty-games-kick.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `declaration-block-no-redundant-longhand-properties` and `shorthand-property-no-redundant-values` false negatives for `inset` shorthand
```

Comment:
```suggestion
Fixed: `*-no-redundant-*` false negatives for `inset` shorthand
```

[Ref](https://stylelint.io/user-guide/rules#redundant) and [conventions](https://stylelint.io/maintainer-guide/issues#titles)